✨ ENH: Auto-detecting the target (#45)
executablebooks/github-activity: docs/index.md

the command-line interface. It takes the following form:

```
github-activity [<org>/<repo>] --since <date or ref> --until <date or ref>
```

The `[<org>/<repo>]` argument is **optional**.

> If you do not give it, then `github-activity` will attempt to infer this value by running `git remote -v` and using either `upstream` or `origin` (whichever it finds first).

Author: ```suggestion
If you do not give it, then `github-activity` will attempt to infer this value by running `git remote -v` and using either `upstream` or `origin` (preferring `upstream` if both are available).
```
